Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM ubuntu:16.04
+
+ADD . /app
+
+RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+
+RUN apt-get update
+RUN apt-get build-dep -y python-imaging
+RUN apt-get install -y python3-pip \
+                       libjpeg8 \
+                       libjpeg62-dev \
+                       libfreetype6 \
+                       libfreetype6-dev \
+                       libpq-dev \
+                       wget
+
+ENV NVM_DIR /usr/local/nvm
+ENV NODE_VERSION 4.2.0
```

Comment:
Ok, I've checked it and it looks that there are some problems with installing outdated npm packages.

To not block this pull request I will create another issue to upgrade all npm dependencies and move to current node LTS.